src/main/scala/mesosphere/marathon/state/Timestamp.scala

```diff
@@ -14,72 +14,71 @@ import scala.math.Ordered
 /**
   * An ordered wrapper for UTC timestamps.
   */
-abstract case class Timestamp private (private val utcDateTime: DateTime) extends Ordered[Timestamp] {
+abstract case class Timestamp private (private val instant: Instant) extends Ordered[Timestamp] {
   def toOffsetDateTime: OffsetDateTime =
     OffsetDateTime.ofInstant(
-      Instant.ofEpochMilli(utcDateTime.toInstant.getMillis),
-      utcDateTime.getZone.toTimeZone.toZoneId)
+      instant,
+      ZoneOffset.UTC)
```

Comment:
Hm, did we always assume UTC before?

Reply:
it seems that we stored `utcDateTime` before, so I think so

Reply:
you can check the deleted line 60:
```scala
def apply(dateTime: DateTime): Timestamp = new Timestamp(dateTime.toDateTime(DateTimeZone.UTC))
```